src/components/user-list.html

```diff
@@ -0,0 +1,25 @@
+<link rel="import" href="./user-profile.html">
+
+<dom-module id="user-list">
+    
+    <template>
```

Comment:
It looks like you have two opening and one closing `<template>` tags here.

Reply:
This one was weird, unless I wrapped the <script> also, it would error out that PolymerElements was not defined. Guessing it has something to do with the <template> and <style> being empty?